tests/test_pytest_plugins.py

```diff
@@ -81,7 +79,7 @@ def test_path_parts(self, input, expected):
     def test_on_py_path(self):
         cwd_parts = _path_parts(py.path.local())
         folder_parts = _path_parts(py.path.local("a/b/c"))
-        assert folder_parts[len(cwd_parts):] == ["a", "b", "c"]
+        assert folder_parts[len(cwd_parts) :] == ["a", "b", "c"]
```

Comment:
wat?

Reply:
@ambv - why is that?

Reply:
reported upstream

Reply:
https://github.com/PyCQA/pycodestyle/issues/373 - black is right it seems

Reply:
Just my thoughts about this seem strange at first (like many other black style formats) but it does make sense once you look into it more in depth.

Reply:
@rpkilby, if you look at the "Yes"/"No" examples below this section, you will see function calls explicitly using a space separating the colon.

Reply:
@ambv the examples use a space, but it doesn't seem to be a part of the recommendation. At least, that's not how the text reads to me.

Regardless, the style black enforces *is* compliant, I just don't happen to like that particular choice. But again, that's just my irrelevant preference.

Reply:
Thanks! I agree that this is what PEP8 describes and I have succesfully been unaware of it for at least a decade, although I have red PEP8 many times. Another big :+1: for @ambv for being the perfectionist he is and getting stuff like this down to the last little detail.